core/schema/src/main/resources/sql/eventconf_events.sql

```diff
@@ -2761,3 +2761,161 @@ INSERT INTO eventconf_events(id, source_id, uei, event_label, description, enabl
 </event>', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, 'system-migration');
 
 ALTER SEQUENCE eventconf_events_id_seq RESTART WITH 158;
+
+UPDATE eventconf_events SET json_content = '{"collectionGroup":[],"filters":[],"eventLabel":"OpenNMS-defined trap event: SNMP_Cold_Start","uei":"uei.opennms.org/generic/traps/SNMP_Cold_Start","mask":{"maskelements":[{"mename":"generic","mevalues":["0"]}],"varbinds":[]},"descr":"<p>A coldStart trap signifies that the sending protocol entity is reinitializing itself such that the\\n            agent''s configuration or the protocol entity implementation may be altered.</p>","logmsg":{"dest":"LOGNDISPLAY","content":"Agent Up with Possible Changes (coldStart Trap)\\n        ","notify":true},"severity":"Normal","autoactions":[],"varbindsdecodes":[],"operactions":[],"loggroups":[],"forwards":[],"scripts":[],"parameters":[]}', last_modified = CURRENT_TIMESTAMP WHERE id = 1;
```

Comment:
You shouldn't update this file.  We never want to break liquibase upgrade even in `develop` branch.

Create a new changeset that handles update and use separate file to have this update.